src/status_im2/contexts/quo_preview/settings/category_reorder.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im2.contexts.quo-preview.settings.category-reorder
```

Comment:
of course, keep other parts in different files if it's cleaner but our catalogue of components should be almost equivalent to look up as the figma design system. Otherwise it adds confusion to what is there/not there etc

Reply:
@J-Son89 done 👍